python/lib/cloudutils/serviceConfig.py

```diff
@@ -617,11 +617,8 @@ def config(self):
             cfo.addEntry("LIBVIRTD_ARGS", "-l")
             cfo.save()
             if os.path.exists("/lib/systemd/system/libvirtd.socket"):
-                bash("/bin/systemctl mask libvirtd.socket");
-                bash("/bin/systemctl mask libvirtd-ro.socket");
-                bash("/bin/systemctl mask libvirtd-admin.socket");
-                bash("/bin/systemctl mask libvirtd-tls.socket");
-                bash("/bin/systemctl mask libvirtd-tcp.socket");
+                bash("/bin/systemctl mask libvirtd.socket libvirtd-ro.socket libvirtd-admin.socket libvirtd-tls.socket libvirtd-tcp.socket");
+                bash("/bin/systemctl mask virtqemud.socket virtqemud-ro.socket virtqemud-admin.socket virtqemud virtnetworkd virtstoraged");
```

Comment:
if making it long lines, why not all on one line. One fork instead of two.
```suggestion
                bash("/bin/systemctl mask \
                libvirtd.socket \
                libvirtd-ro.socket \
                libvirtd-admin.socket \
                libvirtd-tls.socket \
                libvirtd-tcp.socket \
                virtqemud.socket \
                virtqemud-ro.socket \
                virtqemud-admin.socket \
                virtqemud \
                virtnetworkd \
                virtstoraged");
```

Reply:
Saves time and makes execution faster.